packages/web/app/components/Invoice/EditInvoiceModal/InvoiceItem.jsx

```diff
@@ -256,9 +260,17 @@ export const InvoiceItemRow = ({
       productCode: value.code,
       productPrice: value.price,
       productDiscountable: value.discountable,
+      product: { ...item.product, price: value.price },
     });
   };
 
+  useEffect(() => {
+    formArrayMethods.replace(index, {
+      ...item,
+      productPrice: defaultItem?.productPrice,
+    });
+  }, []);
```

Comment:
```suggestion
  }, [defaultItem?.productPrice]);
```
Should this reset everytime the default item changes?